kong/runloop/handler.lua

```diff
@@ -690,143 +690,142 @@ local function _register_balancer_events(f)
 end
 
 
-local function register_events()
-  -- initialize local local_events hooks
-  local db             = kong.db
-  local core_cache     = kong.core_cache
-  local worker_events  = kong.worker_events
-  local cluster_events = kong.cluster_events
-
-  if db.strategy == "off" then
-
-    -- declarative config updates
-
-    local current_router_hash
-    local current_plugins_hash
-    local current_balancer_hash
+local reconfigure_handler
+do
+  local now = ngx.now
+  local update_time = ngx.update_time
+  local ngx_worker_id = ngx.worker.id
+  local exiting = ngx.worker.exiting
+
+  -- '0' for compare with nil
+  local current_router_hash   = 0
+  local current_plugins_hash  = 0
+  local current_balancer_hash = 0
```

Comment:
The PR says that it uses module level variables, but these are really `do block` level variables. Should you move them to module level and capitalize them as:

```
local CURRENT_ROUTER_HASH  = 0
local CURRENT_PLUGINS_HASH = 0
local CURRENT_BALANCER_HASH = 0
```

Though I am not sure if it is worth it, do we need them anywhere else in this module, or do we need to export them as part of this module, e.g. `get_router_hash()` etc. functions?

Reply:
Thanks for you opinions.

I think it is my mistake, these hash vars are not module level, they are `do block` level.
I check the code again and I am sure these hashes are only used in this module.